ci-operator/jobs/stackrox/scanner/stackrox-scanner-master-postsubmits.yaml

```
@@ -453,3 +453,93 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^master$
+    cluster: build02
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
```

I'd bump all postsubmits to 5 or they stack up.
```suggestion
    max_concurrency: 5
```